## 80_security_tools.patch by Mike Hommey <glandium@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Enable building of some NSS tools.
## DP: Disable rpath.

diff --git a/mozilla/security/nss/cmd/platlibs.mk b/mozilla/security/nss/cmd/platlibs.mk
index 65969fc..3c41483 100644
--- a/mozilla/security/nss/cmd/platlibs.mk
+++ b/mozilla/security/nss/cmd/platlibs.mk
@@ -38,6 +38,7 @@
 # set RPATH-type linker instructions here so they can be used in the shared
 # version and in the mixed (static nss libs/shared NSPR libs) version.
 
+ifdef ENABLE_RPATH
 ifeq ($(OS_ARCH), SunOS) 
 ifeq ($(BUILD_SUN_PKG), 1)
 ifeq ($(USE_64), 1)
@@ -65,6 +66,7 @@ EXTRA_SHARED_LIBS += -Wl,-rpath,'$$ORIGIN/../lib'
 endif
 endif
 endif
+endif
 
 ifeq ($(OS_ARCH), HP-UX) 
 ifeq ($(OS_TEST), ia64)
